### PR TITLE
Dockerized app

### PR DESCRIPTION
**Pull request by jakkemomo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
 # Meetups App
 
-## First run
-1. Create `.env` file in the root dir with fields from `.env.example`
-2. Optionally (to use maps) [get and set](https://yandex.ru/dev/jsapi-v2-1/doc/ru/#get-api-key) yandex maps api key or, even better, [google maps api key](https://developers.google.com/maps/documentation/javascript/get-api-key) in the `.env` file
-3. Install docker-compose
-4. `chmod +x entrypoint.sh && ./entrypoint.sh` (You can also use entrypoint-mac.sh if you are using mac, but be sure to install psycopg lib before that)
+## Basic setup
+1. Create `.env` file in the root dir with filled out fields from `.env.example`
+2. Install docker-compose for setting up local postgres db and application itself
```
Comment (sakalex) [inline README.md:5]: Let's move docker and docker -compose to the Requirement section

File: README.md [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
-This will install postgres db in docker, install geo dependencies, create superuser admin:admin, run migrations and run server.
-5. After you have done this once you can just run `python manage.py runserver` next time
+## First run for local development (will install dependencies locally)
+1. Create `.env` file in the root dir with filled out fields from `.env.example`
+2. Install docker-compose for setting up local postgres db
+3. Run your postgres db via `docker-compose -f deployments/docker-compose-db.yml up -d`
+4. For linux: install geo dependencies via `sudo apt-get install binutils libproj-dev gdal-bin`
+5. For mac: install geo dependencies via `brew install binutils proj gdal`
+6. Setup your virtual environment and install requirements via `pip install -r requirements/local.txt`
+7. Install pre-commit hooks via `pre-commit install` for code formatting and linting before commit
```
Comment (sakalex) [inline README.md:16]: Looks like "prep-steps" or "project setup"

File: README.md [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
-4. `chmod +x entrypoint.sh && ./entrypoint.sh` (You can also use entrypoint-mac.sh if you are using mac, but be sure to install psycopg lib before that)
+## Basic setup
+1. Create `.env` file in the root dir with filled out fields from `.env.example`
+2. Install docker-compose for setting up local postgres db and application itself
+3. Run `docker-compose -f deployments/docker-compose.yml up -d`
+4. Go to `http://localhost:8000/swagger` and you should see the app running
 
-This will install postgres db in docker, install geo dependencies, create superuser admin:admin, run migrations and run server.
-5. After you have done this once you can just run `python manage.py runserver` next time
+## First run for local development (will install dependencies locally)
```
Comment (sakalex) [inline README.md:9]: My suggestion is to completely do away with local launching not in docker
And not even to describe such steps in order not to confuse young developers.

File: deployments/docker-compose.yml
Comment (sakalex) [inline deployments/docker-compose.yml:1]: Missing blank line at the end of the file

File: deployments/app/scripts/run.sh
Comment (sakalex) [inline deployments/app/scripts/run.sh:1]: Missing blank line at the end of the file

**Review by sakalex — COMMENTED** [review]
approve with minor comments

File: deployments/docker-compose.yml [hunk 4]
```diff
@@ -3,21 +3,35 @@ services:
...
+
+  app:
+    container_name: app
+    build:
+      context: ../
+      dockerfile: deployments/app/Dockerfile
+    restart: always
+    command: /app/deployments/app/scripts/run.sh
+    volumes:
+        - ../:/app
```
Comment (sakalex) [inline deployments/docker-compose.yml:25]: Why you use /app for mount if in Dockerfile we using /home/app/web as home dir for web app?

File: README.md (hunk 3 above)
Comment (jakkemomo) [reply README.md:9]: I think that the main thing they should learn is how to **debug** the app. And it is much easier to debug a local python process than a process inside a running container, especially if you are a young padawan

File: README.md (hunk 2 above)
Comment (jakkemomo) [reply README.md]: Yes, however it seems like these procedures go together, and there is really no reason to separate them from the local setup at this point

**Pull request merged**